src/imports.rs

```diff
@@ -44,8 +44,7 @@ impl<'a> FmtVisitor<'a> {
             Some(item.vis.clone()),
             Some(item.span.lo()),
             Some(item.attrs.clone()),
-        )
-        .rewrite_top_level(&self.get_context(), shape);
+        ).rewrite_top_level(&self.get_context(), shape);
```

Comment:
This (and other similar changes) should not happen. You could be using older rustfmt.
```suggestion
        )
        .rewrite_top_level(&self.get_context(), shape);
```